src/pages/authorization/authorizing-api-calls/index.mdx

```diff
@@ -793,6 +793,16 @@ To achieve that, make the same exact same request as in [Case: New customer](#ca
 
 Your `entity_id`, or any other customer identity data, won' be overwritten. The web browser will attach cookies to the server allowing it to recognize you as an existing customer. You'll get a new access token, and cookies' lifetime will be extended, too. They will be valid for 2 years from your last request to `https://accounts.livechat.com/customer/token`.
 
+
+### Case: Existing customer chatting in different groups
```

Comment:
This title is misleading for me - maybe sth like this: Customer chatting in different groups with separate identities or sth like that? @ale-kacperczyk wdyt? Also, this refers to both: new customer and existing customer.

Reply:
One more correction from us